feature/policy_forwarding/encapsulation/otg_tests/staticgueencap_and_bgp_path_selection/README.md

```diff
@@ -0,0 +1,745 @@
+| Flow-Set#5 from ATE2:Port1 to ATE1:Port1 are sent Unencaped                                          | `$ATE2_INTERNAL10.v6/64 --> $ATE1_PORT1_user5.v6/64` | AF4  |                             |
+
+
+### DUT Configuration:
+
+In addition to the adjacencies and peering configurations described in the
+tables above, the DUT requires the following configurations:
+
+*   **IS-IS:**
+
+    *   The DUT's loopback interface must be passive for IS-IS.
+
+*   **BGP:**
+
+    *   Define import and export route policies to match the advertisements for
+        each BGP peering.
+
+*   **Static GUE Encapsulation:**
+
+    *   Configure static GUE encapsulation as follows:
+
+        *   Define UDP ports to be used for IPv4oUDP and IPv6oUDP which is 6080.
+        *   Define the Tunnel NHG configuration with these parameters:
+
+            *   `ttl = 128`
+            *   `tunnel-source = $DUT_lo0.v4`
+            *   `tunnel-destination1 = $ATE2_INTERNAL_TE11.v4/32`.
+            *   `tunnel-destination2 = $ATE2_INTERNAL_TE10.v4/32`.
+            *   The DUT must have a static route pointing `"$ATE2_PPNH1.v6/128"`
+                and `$ATE2_PPNH2.v6/128` to the NHG created above (below is an
+                example of the static route). The IBGP peer $ATE2_C.IBGP.v6/128
+                is expected to advertise both IPv4 and IPv6 prefixes with the
+                Next-Hop as `"$ATE2_PPNH1.v6/128"` or `"$ATE2_PPNH2.v6/128"`
+                `static dst: ATE2_PPNH1.v6/128 next-hop:
+                $ATE2_INTERNAL_TE11.v4/32 static dst: ATE2_PPNH2.v6/128
+                next-hop: $ATE2_INTERNAL_TE10.v4/32`
+
+*   **GUE Decapsulation:**
+
+    *   For a GUE decapsulation node, configure the following:
+
+        *   UDP port 6080 (configurable) must be used for decapsulating IPv4 and
+            IPv6 payload. The implementation MUST look at the first 4 bits of
+            the UDP payload to determine the GUE version as well as the IP
+            version of the payload to be either IPv4 or IPv6 as explained in the
+            [IETF draft](https://datatracker.ietf.org/doc/html/draft-ietf-intarea-gue-09#section-4)
+        *   The Decapsulation node must be configured for decapsulating traffic
+            received for the range i.e $DUT_TE11.v4/30 and $DUT_TE10.v4/30 in
+            place of the corresponding /32 addresses.
+        *   After decapsulation, the outer TTL and DSCP bits must not be copied
+            to the inner header
+
+### Following Health checks to be run Pre and Post every sub test
+
+Use # Health-1.1: Generic Health Check. If errors identified then the test Must
+fail.
+
+## RT-3.52.1: Baseline
+
+*   **Test Steps:**
+
+    *   Generate DUT Configuration as specified in the test setup
+    *   Use gnmi to push the configuration to the DUT
+    *   Ensure no prefixes are exchanged over the IBGP peering between
+        `$ATE2_C.IBGP.v6` and `$DUT_lo0.v6`. Validate this using OC
+        ```
```

Comment:
The use of the triple backtick has to be on the 1st column, otherwise you get unpredictable results.  Notice how much of the remainder (but not all) is treated as static text.  

Convert all the triple back ticks to start and end on the 1st column and check the mdformat results to make sure this looks as intended.   

Right now major sections of the doc are getting excluded as static /codeblock text which is causing validation to fail for ocpaths and canonical_oc

Reply:
Done